test/it/client-listUsers.js

```diff
@@ -97,16 +112,16 @@ describe('client.listUsers().each()', () => {
         setTimeout(reject.bind(null, 'foo error'), 1000);
       });
     }).catch((err)=>{
-      assert.equal(localCount, 1);
-      assert.equal(err, 'foo error');
+      expect(localCount).to.equal(1);
+      expect(err).to.equal('foo error');
     });
   });
 });
 
 describe('client.listUsers().next()', () => {
   it('should return User models', () => {
     return client.listUsers().next().then(result => {
-      assert.instanceOf(result.value, models.User);
+      expect(result.value).to.be.an.instanceof(models.User);
```

Comment:
in a follow-up PR, it'd be nice to convert all of these to async anonymous functions to match the other tests